groupibthreatintelligenceandattribution_connector.py

```diff
@@ -14,7 +14,7 @@
 from dateparser import parse
 from phantom.action_result import ActionResult
 from phantom.base_connector import BaseConnector
-from pytia import TIAPoller
+from cyberintegrations import TIPoller
```

Comment:
Hi @LanskoyGIB
Can you move this to Line#14, before import of dataparser?
This will help to pass the Lint.

Reply:
@ishans-crest, Hello!
Your recommendation helped us a lot, thanks, but we got another error with blank spaces, after reading PEP8 documentation, i still cannot understand where exactly error with blank spaces,
Thank in advance